### DISCUSSION TITLE
Use text-#****** or bg-#****** as class names to change the color of text or the color of the background.

### DISCUSSION THREAD

**Discussion by Junvary** [quasarframework/quasar#15703] [Ideas / Proposals]
example:
```
<!-- changing text color -->
<p class="text-#a2aa33">...</p>

<!-- changing background color -->
<p class="bg-#a2aa33">...</p>
```

**Comment by dongwa** [comment]
Quasar has not provided this CSS now. You can install an atomic CSS library in your Quasar project to get this ability.
Such as [unocss](https://unocss.dev),[use unocss in quasar](https://github.com/unocss/unocss/blob/main/examples/quasar/quasar.config.js) or [tailwindcss](https://tailwindcss.com/)

And then,We can see that Quasar has a roadmap to use UnoCSS as the CSS engine in Quasar 3, which might enable this ability in Quasar 3.[See](https://github.com/orgs/quasarframework/projects/2?pane=issue&itemId=1451670)
